### ISSUE TITLE
instrumentation-bunyan / instrumentation bug "_this.enable is not a function"

### ISSUE THREAD

**Issue by AkselAllas** [open-telemetry/opentelemetry-js-contrib#1205]
[Codesandbox repro](https://codesandbox.io/s/optimistic-lena-idtbvr?file=/src/Observability.ts)

<img width="1257" alt="image" src="https://user-images.githubusercontent.com/26136082/192511386-b7bce781-8e37-4073-9735-8d05f000df58.png">

Interesting fact is that the object we pass to register instrumentations does have `enable`
<img width="933" alt="image" src="https://user-images.githubusercontent.com/26136082/192511584-fe4f2236-c040-4b4a-91aa-397462261592.png">

Stacktrace, where the code fails
<img width="1001" alt="image" src="https://user-images.githubusercontent.com/26136082/192510596-9b655c9b-40a2-48d7-b5d6-7cf86e9faa03.png">

Problem file in question 
![image](https://user-images.githubusercontent.com/26136082/192510658-e328bc9c-b44f-461c-ac4a-c12fd9af86e6.png)

**Comment by Flarna** [comment]
Not sure here but the error message somewhat indicates a version mixmax:
```
InstrumentationBase@https://idtbvr.csb.app/node_modules/
opentelemetry/instrumentation/0.32.0/build/esm/platform/browser/instrumentation.js:42:13
```
your package.json says 0.33.0.

Maybe some leftover after upgrade? Please try a fresh install after deleting node_modules and package-lock.json.

At least I don't see this error in a similar local sample.

**Comment by AkselAllas** [comment]
I did `yarn cache clean`, deleted yarn.lock and deleted node_modules, did fresh install locally. Made sure I got instrumentation `v0.33` and it worked 👍 🤔 

I tried downgraded versions in codesandbox. But that doesn't work. Some weird setup makes codesandbox not get instrumentation 0.33 when asked for it.

**Comment by AkselAllas** [comment]
I'm an idiot.

**Comment by AkselAllas** [comment]
@Flarna

**Comment by AkselAllas** [comment]
I went through all dependencies. I had `@opentelemetry/core` `@1.4.0` `@1.6.0` and `@1.7.0`. I made sure I only had `@1.7.0` but didn't help.

I also have a source map error, even though I don't think it should do anything.

```
Failed to parse source map from '/Users/akselallas/Dev/test/node_modules/@opentelemetry/instrumentation-bunyan/src/instrumentation.ts' file: Error: ENOENT: no such file or directory, open '/Users/akselallas/Dev/testnode_modules/@opentelemetry/instrumentation-bunyan/src/instrumentation.ts'
```

The real path for the sourcemap is `@opentelemetry/instrumentation-bunyan/build/src/instrumentation.ts`

**Comment by AkselAllas** [comment]
The real problem is that the `instrumentation.js` in the browser version of node_modules has

```
export abstract class InstrumentationAbstract<T = any> implements types.Instrumentation {
...
  public abstract enable(): void;
}
```

**Comment by Flarna** [comment]
Yes, seems update of modules in contrib to use 0.33.0 from core is not yet done. `^0.32.0` doesn't use 0.33.0 so to avoid duplicates of `@opentelemetry/instrumentation` it's needed to stick on 1.6.0/0.32.0 of OTel core until contrib components are updated and published.

`InstrumentationAbstract` is not browser/node specific, it's a common base class. `InstrumentationBase` is different.

**Comment by AkselAllas** [comment]
Figured out why it doesn't work. It's simply not supported and not supposed to work for browser.

More specifically `opentelemetry-instrumentation` of `opentelemetry-js` has a browser implementation of `InstrumentationBase` with no written `enable` method. 
`(experimental/packages/opentelemetry-instrumentation/src/platform/browser/instrumentation.ts)`

[15 line skeleton class for `browser`](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation/src/platform/browser/instrumentation.ts) while [180 line proper class for `node`](https://github.com/open-telemetry/opentelemetry-js/blob/main/experimental/packages/opentelemetry-instrumentation/src/platform/node/instrumentation.ts)

I tried using the node implementation for browser, but it doesn't work, as it uses node specific `require` / `fs` etc packages. And I wound up with 19 polyfill errors ❤️
